core/src/main/scala/no/nrk/bigquery/BigQueryClient.scala

```diff
@@ -558,9 +558,9 @@ class BigQueryClient[F[_]](
         }
     }
 
-  def getRoutine(udfId: UDF.UDFId.PersistentId): F[Option[Routine]] =
+  def getRoutine(persistentId: BQPersistentRoutine.PersistentRoutineId): F[Option[Routine]] =
```

Comment:
Not a fan of these excessively long names.
`BQPersistentRoutine.PersistentRoutineId` => `BQPersistentRoutine.Id`
namespacing is done via the object.

Reply:
Sure, no objection on shortening the names as long as we're including the object name:-)